dist/pinny.js

```diff
@@ -467,6 +470,26 @@
             return $(document.activeElement);
         },
 
+        _isWithin: function($element, selector) {
+            return ($element.closest('.' + selector).length) ? true : false;
```

Comment:
@scalvert the way you write it I would have to call the method 
```
isChildOf('.' + classes.PINNY) 
```
do you mind if I re-write the function like this:
```
isChildOf: function(selector) {
            return $(this).parents().filter(function () {
                return $(this).hasClass(selector);
            }).length > 0;
        }
```
so that I'll be calling ```$(element).isChildOf(classes.PINNY)```, which is more consistent with the rest of the code?

Also, would it be worth it to have a separate utility file for these types of custom Zepto/jQuery extensions? Or for now these functions should just go at the bottom of the file (before the initialization code)?